.nostr.local/settings.json

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
Not a fan of having to maintain multiple of these. I'll come up with some mechanism to `extend` a default settings file. No action needed here.

Reply:
Tracked here: https://github.com/Cameri/nostream/issues/119